contents/blog/soft-delete/index.mdx

```diff
@@ -0,0 +1,255 @@
+---
+title: Using the New Soft-delete Feature
+date: 2022-10-17
+author: Maciej Obuchowski & Michael Robinson
+image: ./image.svg
```

Comment:
On the first blog post the image was 598x266. That's a somewhat random aspect ratio, admittedly.

The two should match, I think. Perhaps we can pick an easier to remember ratio (like 600x270) and make both posts' images match?